File: backend/core-api/src/main/java/io/basearchitecture/core/dto/sample/SampleItemPreparationRequest.java
package io.basearchitecture.core.dto.sample;

import jakarta.validation.Valid;
import jakarta.validation.constraints.NotNull;

/** 예제 준비 API의 공개 요청 DTO입니다. */
public record SampleItemPreparationRequest(
        @NotNull @Valid SampleItemRequest item
) {
}

File: backend/core-api/src/main/kotlin/ai/govbiz/core/_sampleitem/dto/SampleItemPreparationRequest.kt
package ai.govbiz.core._sampleitem.dto

import jakarta.validation.Valid
import jakarta.validation.constraints.NotNull

/** 예제 준비 API의 공개 요청 DTO입니다. */
data class SampleItemPreparationRequest(
    @field:NotNull
    @field:Valid
    val item: SampleItemRequest?,
)
